lib/sidekiq/cron/launcher.rb

```diff
@@ -1,3 +1,4 @@
+require 'sidekiq/cron'
```

Comment:
Is this require necessary? I think we can ✂️

Reply:
Won't we need it to use `Sidekiq::Cron.configuration.cron_poll_interval.to_i` in `initialize`?

Reply:
But in the entry point `lib/sidekiq-cron.rb` of the library we already load all the modules:

```ruby
require "sidekiq"
require "sidekiq/cron"
require "sidekiq/cron/job"
require "sidekiq/cron/namespace"
require "sidekiq/cron/poller"
require "sidekiq/cron/launcher"
require "sidekiq/cron/schedule_loader"
```

So when this file is required, the `sidekiq/cron` was loaded too, right?

Reply:
In that case, how about removing below require from `lib/sidekiq/cron/job.rb`?
```
require 'sidekiq'
require 'sidekiq/cron'
require 'sidekiq/cron/support'
```